.github/workflows/aws-deploy.yml

```
@@ -0,0 +1,95 @@
+---
+name: AWS Deployment
+
+on: 
+  workflow_dispatch:
+    inputs:
+        environment:
+            description: "Environment to deploy to"
+            default: "staging"
+            options:
+                - staging
+                - production
+            required: true
+            type: choice
+       
+env:
+  AWS_ACCOUNT_ID: ${{ vars.AWS_PUBLIC_DATA_RELEASES_ACCOUNT_ID }}
+  AWS_REGION: ${{ vars.AWS_DEFAULT_REGION }}
+  STAGING_CLOUDFRONT_DISTRIBUTION_ID: ${{ secrets.STAGING_CLOUDFRONT_DISTRIBUTION_ID }}
+  STAGING_S3_BUCKET: s3://staging.volumeviewer.allencell.org
+  PRODUCTION_CLOUDFRONT_DISTRIBUTION_ID: ${{ secrets.PRODUCTION_CLOUDFRONT_DISTRIBUTION_ID }}
+  PRODUCTION_S3_BUCKET: s3://volumeviewer.allencell.org
+
+permissions:
+    id-token: write  # Required for requesting the JWT and OIDC
+    contents: write  # Required for actions/checkout and OIDC tokens
+
+jobs:
+    build:
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout
+          uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
+  
+        - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8
+          with:
+            node-version: "18"
+  
+        - name: Install and Build
+          run: npm ci && npx vite build
```

```suggestion
          run: npm ci && npm run gh-build
```
(Minor difference from nucmorph colorizer's build command)